index.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Hiwot's Portfolio</title>
+ 
+<link rel="stylesheet" href="styles.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Inter">
+</head>
+
+<body> 
+    <header class="navigation">
+        <nav class="toolbar-container">
+            <div class="logo"><a href="#welcome">Welcome</a></div>
+            <div>
+                <a class="icon" ><a href="#menu-bar">
+                <i class="fa fa-bars"></i></a>
+            </div>
+        </nav>
+    </header>
+    <section class="headline back-image">
+        <div class="primary-headline" id="desktop-primary">
+            <h1>Hey there. I'm <br> Hiwot<span class="h1color"><br>
+             I'm a software developer</span>
+            </h1>
+        </div>
+         <div>
+            <p class="supporting-text">  I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+        </div>
+        <div>
+            <ul class="socialt">
+                <li> <a href="#"><img src="image/twitter.svg" /></a> &nbsp; &nbsp; </li>
+                <li> <a href="#"><img src="image/vector.png" /></a> </li>
+                <li> <a href="#"><img src="image/Vector (1).png" /></a> &nbsp; &nbsp; </li>
+                <li> <a href="#"><img src="image/medium.svg" /></a> &nbsp; &nbsp; </li>
+                <li> <a href="#"><img src="image/github.svg" /></a> &nbsp; &nbsp; </li>
+                <li> <a href="#"><img src="image/angellist.svg" /></a> &nbsp; </li>
+            </ul> 
+        </div>
+        <div class="scroll">
+            <img src="image/Disabled.png" alt="Scroll down" >
+        </div>
+    </section>
+    <section class="recent-works-container">
+        
+         <div class="primary-text">
+            <div><h3> My Recent Works</h3></div>
+             
+             <div><img src="image/indicator.jpg" class="indicator"></div>
+        </div>
+         <section class="projects">
+            <div class="project1">  
+                
+                <div class="project-detail">
+                    <h4>Multi-Post Stories Gain+Glory</h4>
+                    <ul class="button-detail"> 
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavScript</li>
+                        <li>html</li>
+                    </ul>
+                    <button class="see-project">See project </button>
```

Comment:
- [ ] Great job styling the button and adding the button states. However, I have realized that the colors used do not really match the ones provided in the Figma design for the different states. Below I will include a screenshot of the correct button states colors as provided in the Figma design.

![image](https://user-images.githubusercontent.com/86318284/198611671-43c6b4a1-2f69-4830-a71d-c1695d44c3ee.png)

**Note:** I have highlighted just this one button. However, all the buttons in your project, the ones in the Works section and the about section, all follow the same pattern. It would be redundant to re-highlight all of them. This said, kindly ensure that you make this change on all the existing buttons in your project.